lib/provider.js

```diff
@@ -31,27 +49,30 @@ module.exports = {
       return this.scanFeaturesDir(results)
     }
   },
-  scanStepDefinitionsDir: function(results) {
+  scanStepDefinitionsDir: function(results = []) {
     //TODO: first search step definitions for your file
-    for (var step_def_file of fs.readdirSync(`${this.rootDirectory()}${this.featuresDirectory()}/step_definitions`)) {
+    const searchPath = `${this.rootDirectory()}${this.featuresDirectory()}/step_definitions`;
+    const allStepPaths = this.searchForPattern(searchPath, /\.js.*$/);
```

Comment:
I don't think we can use `.js` here as the package should be language agnostic. For example, I use this plugin on Rails projects